mkosi/backend.py

```diff
@@ -183,14 +183,7 @@ def detect_distribution() -> tuple[Optional[Distribution], Optional[str]]:
 
 
 def is_rpm_distribution(d: Distribution) -> bool:
-    return d in (
```

Comment:
This should probably just be renamed to `is_dnf_distribution()` since that's what it is used for